dashboard/app/views/home/_levelbuilder.html.haml

```diff
@@ -5,7 +5,7 @@
       = link_to 'Scripts', scripts_path
       = link_to '(Update all)', scripts_path(rake: '1'), data: { confirm: 'Are you sure want to re-seed script data?'}
     %li= link_to t('builder.manage'), levels_path
-    %li= link_to 'Pools', pools_path
+    %li= link_to 'Block Pools', pools_path
```

Comment:
Unrelated, but hopefully this will help some future levelbuilder looking at the admin menu for the first time and being like "what are pools??"